### ISSUE TITLE
Upgraded from 5.3.4 to 5.4.5 and enablessl and groups do not seem to be working

### ISSUE THREAD

**Issue by rshmhrj** [e2guardian/e2guardian#753]
Upgraded to `jammy_V5.4.5r_20220907.deb` on Ubuntu 22.04.1 LTS.  I have a simple Squid running (no ICAP) and things were pretty smooth.  After upgrading starting e2guardian service fails without any clear reasons -- it just says `exit-code`.

```
/etc/e2guardian ᐅ systemctl status e2guardian.service
× e2guardian.service - E2guardian Web filtering
     Loaded: loaded (/lib/systemd/system/e2guardian.service; enabled; vendor preset: enabled)
     Active: failed (Result: exit-code) since Thu 2022-11-17 02:00:01 PST; 1min 27s ago
    Process: 45053 ExecStart=/usr/sbin/e2guardian (code=exited, status=1/FAILURE)
        CPU: 6ms

Nov 17 02:00:01 ubuntu-X220T systemd[1]: Starting E2guardian Web filtering...
Nov 17 02:00:01 ubuntu-X220T systemd[1]: e2guardian.service: Control process exited, code=exited, status=1/FAILURE
Nov 17 02:00:01 ubuntu-X220T systemd[1]: e2guardian.service: Failed with result 'exit-code'.
Nov 17 02:00:01 ubuntu-X220T systemd[1]: Failed to start E2guardian Web filtering.
```

The error goes away and e2guardian starts if I comment out the `enablessl=on` line in `e2guardian.conf`.

I also took some time to set up two filtergroups (group1 and group2, with ubuntu usernames inside `/etc/e2guardian/lists/authplugins/filtergroupslist` (e.g. `rishi=2`).  I wanted to have tight blocking on the default group1 and no restrictions at all on group2.

No matter which group I put myself under, it seems to only run under the default group.

I backed up the configs and deleted the folder and reinstalled, but most of the default configs did not get created.  I manually re-added the configs one by one until it came up again.  (Is there away to download or make a clean set of default configs from src -- simply copying the `configs` folder isn't sufficient because the names are suffixed with `.in` and I'm not sure how to run the Makefiles correctly or what setup is needed).

The `enablessl` and being routed through the default group issues still persist after reinstalling and re-setting up -- not sure how to continue or what commands to run to get logs aside from what I pasted above.  Any thoughts or suggestions would be greatly appreciated!

**Comment by philipianpearce** [comment]
Have you read and followed the instructions found here?  https://github.com/e2guardian/e2guardian/blob/v5.4/notes/Upgrading_to_V5.4

Also, check your system log /var/log/messages for errors output from e2guardian - should tell you why it has failed.

**Comment by philipianpearce** [comment]
Also,   this is not a bug so should be asked on google group see #21

**Comment by rshmhrj** [comment]
Hi @philipianpearce thanks for the quick response and references!  I did follow the suggestions from the upgrade guide when performing it.  Took a look at it again to see if there was anything that I missed.

I will post the question on the Google group.  Also, FYI re: the system log -- there is no file`/var/log/messages`, though I did check `/var/log/syslog`, but there were no additional details in there.

Thanks again for your help!
